app-service-deploy/entrypoint.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+set -e
+
+LINUX_APP_SUBSTRING='linux'
+
+if [[ -z $WEB_APP_NAME ]];
+then 
+    echo "Required Web App Name. Provide value in WEB_APP_NAME variable" >&2
+    exit 1
+fi
+echo "Web App Name: ${WEB_APP_NAME}"
+
+if [[ -z $RESOURCE_GROUP_NAME ]];
+then 
+    RESOURCE_GROUP_NAME=`az resource list -n "${WEB_APP_NAME}" --resource-type "Microsoft.Web/Sites" --query '[0].resourceGroup' | xargs`
+    if [[ -z $RESOURCE_GROUP_NAME ]];
+    then
+        echo "Web App '${WEB_APP_NAME}' should exist before deployment." >&2
```

Comment:
This check will be missed when the resource group name is provided by the user

Reply:
currently, removed accepting RG, as we dont support ASE.